puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/ansible.groovy

```diff
@@ -23,3 +23,24 @@ def runPlaybook(args) {
         sh "${command.join(' ')}"
     }
 }
+
+def containerPlaybook(playbook) {
```

Comment:
Can ya give a little more breakout and why move this into Ansible library since it's quite container specific?